packages/ui/test/accounts/hooks/useHasRequiredStake.test.tsx

```diff
@@ -0,0 +1,200 @@
+import { cleanup } from '@testing-library/react'
+import { renderHook } from '@testing-library/react-hooks'
+import BN from 'bn.js'
+import React, { ReactNode } from 'react'
+
+import { useHasRequiredStake } from '@/accounts/hooks/useHasRequiredStake'
+import { AccountsContextProvider } from '@/accounts/providers/accounts/provider'
+import { Account, AddressToBalanceMap, Balances } from '@/accounts/types'
+import { ApiContext } from '@/common/providers/api/context'
+
+import { alice, aliceStash, bob } from '../../_mocks/keyring'
+import { MockKeyringProvider } from '../../_mocks/providers'
+import { stubApi } from '../../_mocks/transactions'
+
+const useAccounts: { hasAccounts: boolean; allAccounts: Account[] } = {
+  hasAccounts: true,
+  allAccounts: [],
+}
+
+jest.mock('../../../src/accounts/hooks/useAccounts', () => {
+  return {
+    useAccounts: () => useAccounts,
+  }
+})
+
+let balance: Balances | null = null
+
+const useBalance = {
+  useBalance: () => balance,
+}
+
+jest.mock('../../../src/accounts/hooks/useBalance', () => useBalance)
+
+let balances: AddressToBalanceMap = {}
+
+const useBalances = {
+  useBalances: () => balances,
+}
```

Comment:
This hook is not used in tests, and the `balances` can be inlined AFAICS.

Reply:
Wrong line, I'm guessing you meant `useBalance`